tutorials/functional-full-stack/purescript-bridge/tutorial.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: Connecting a Haskell Backend to a PureScript Frontend
-published: not-yet
+published: 2017-07-17
```

Comment:
Hakyll was choking on this. Also Cynthia is trying to get it published this day, so I'll give her that.